controller/openebs/cstor.go

```diff
@@ -549,39 +550,69 @@ func (p *Planner) isCSISupported() (bool, error) {
 	if err != nil {
 		return false, errors.Errorf("Unable to find kubernetes version, error: %v", err)
 	}
+
+	// compare the kubernetes version with the supported version of csi.
+	comp, err = compareVersion(k8sVersion, types.CSISupportedVersion)
```

Comment:
How is this different from ```isCSISupported, err := p.isCSISupported()``` on line no 513 ?

Reply:
This line is inside the isCSISupported function, right? It is just checking if the k8s version is greater than 1.14.0 or not bcz below that we do not support CSI components installation.

Reply:
Please let me If I didn't understand the question properly @avishnu.

Reply:
Ok, its a part of the same function. Got it.